scripts/memory/__init__.py

```diff
@@ -1,4 +1,5 @@
 from memory.local import LocalCache
```

Comment:
Just a coding convention question for my learning. Is this project following a convention where __init__.py files will contain code? I understand that the standard convention is for `__init__.py files to be empty. Not a criticism, just a question since there are valid use cases for including code in these files.